pkg/crc/oc/oc.go

```diff
@@ -81,7 +81,7 @@ func UseOCWithSSH(sshRunner *ssh.Runner) Config {
 		OcExecutablePath: "oc",
 		KubeconfigPath:   "/opt/kubeconfig",
 		Context:          constants.DefaultContext,
-		Cluster:          constants.DefaultName,
+		Cluster:          constants.DefaultClusterName,
```

Comment:
seeing https://github.com/crc-org/crc/blob/233df0ee1c05f91d49dbe72dc58a2288ac09f30f/pkg/crc/machine/start.go#L556-L559, this could be further improved in follow-up commits.